step-by-step.md

```diff
@@ -127,18 +127,16 @@ So we need to add a `tsconfig.json` configuration file to configure the right la
 }
 ```
 
-> **Note:** when you use the `@sapui5/ts-types-esm` (or `@openui5/ts-types-esm`) types instead, you need to add the following section to tsconfig.json:
+> **Note:** when you use the `@sapui5/types` (or `@openui5/types`) types instead, you need to add the following section to tsconfig.json:
 >
 > ```json
 >        "typeRoots": [
->            "./node_modules/@types"
->        ],
->        "types": [
->            "@sapui5/ts-types-esm"
+>            "./node_modules/@types",
+>            "./node_modules/@sapui5/types"
 >        ],
 >```
 >
-> Why? TypeScript automatically finds all type definition files in a dependency starting with `@types/...` (i.e. all `.d.ts` files in `node_modules/@types/...`). The jQuery d.ts files are there and found, but the SAPUI5 types are only available in a package starting with `@sapui5/...`, hence TypeScript must be explicitly pointed to these types. As this disables the automatic loading of other types from `node_modules/@types/...`, this path must be given as a type root.
+> Why? TypeScript automatically finds all type definition files in a dependency starting with `@types/...` (i.e. all `.d.ts` files in `node_modules/@types/...`). The jQuery d.ts files are there and found, but the SAPUI5 types are only available in a package starting with `@sapui5/...`, hence TypeScript must be explicitly pointed to these types. As this disables the automatic loading of other types from `node_modules/@types/...`, this path must also be given as a type root.

```

Comment:
Hm, after removing the "types" section, this sentence does no longer apply. However, I think the current recommendation is using `types`, not `typeRoots`. Peter?